source/js/modules/Modal.js

```diff
@@ -1,203 +1,230 @@
-/* eslint-disable linebreak-style */
-/* eslint-disable camelcase */
-/* eslint-disable no-undef */
-/* eslint-disable no-underscore-dangle */
-/* eslint-disable no-unused-vars */
-/* eslint-disable indent */
 /* eslint-disable max-classes-per-file */
+
+/** ModalContainer holds and controls all the text necessary for the modal-box.
+ *
+ * Object responsible for holding and parsing text from the given html text-containing element.
+ */
 class ModalContainer {
-    /**
+  /**
      *
-     * @param {HTMLElement} textContainer
+     * @param {HTMLElement} textContainer - The html element that contains the text that we would like to parse into our modal-box
      */
-    constructor(textContainer) {
-        if (!textContainer) throw new Error('[ModalContainer] >>> {../ctor} >>> Object reference resulted in a nullptr');
-        else this.container = textContainer;
+  constructor(textContainer) {
+    // ctor is a shorthand for constructor
```

Comment:
isn't it easier just to use `constructor` everywhere rather than having to explain what an abbreviation means?